README-wiki-path.md

```diff
@@ -0,0 +1,129 @@
+### First of all, please make sure you did the post install path exports
+
+In the example below, ubuntu 16.04 LTS is used with cuda 9.0 (deb install) ,
+cudnn 7.1.4 (deb install), tensorrt 3.0.4 (deb install), as well as bazel 0.17.2 (.sh file install), but other linux distributions with nvidia tar install are possible too
+
+The settings below have been tested to be working and to fix most common path issues, and
+are shown as an interactive help :
+
+
+- a) after cuda 9.0 deb install, it is needed to do :
+
+`export PATH=/usr/local/cuda-9.0/bin${PATH:+:${PATH}} && sudo reboot`
```

Comment:
`PATH` will be reset after `reboot`, what's the meaning of this step?

Reply:
i accidentally deleted my answer...

but to make it short : 

- a reboot is needed to finalize nvidia cuda install
- after the first reboot, `nvcc --version` does not work because the post install action of adding a path has not been done yet
- after `export` is done, and while in the same boot (no reboot yet), now` nvcc --version` will work
- however, after reboot, the `export` will be lost and you need to do `export` again, at every boot
- the solution to this is to add a permanent path with `bashrc`

like that, `nvcc --version` will always work

export

Reply:
I think `export` before `reboot` is unnecessary, just remove it from this step.

Reply:
yes, i was going to do it

Reply:
update : i added all the needed data to my personal github : https://github.com/wonderingabout/nvidia-archives/blob/master/README.md 

i will edit the old file soon to make it simpler and more efficient

Reply:
finished editing, please review again : 

https://github.com/wonderingabout/PhoenixGo/blob/faqv2-bazel-master/docs/path-errors.md